plugins/org.jboss.tools.openshift.express.core/src/org/jboss/tools/openshift/express/internal/core/server/OpenShiftServerExtendedProperties.java

```diff
@@ -24,6 +24,10 @@ public OpenShiftServerExtendedProperties(IAdaptable adaptable) {
 		super(adaptable);
 	}
 	
+	public boolean allowConvenienceEnhancements() {
```

Comment:
Yes, that is really our name for it.  Server Extended Properties was never really meant to be external API. In reality this method shouldnt even be on the extended properties because its more of a UI customization and has nothing to do with the server itself specifically. This method was only added to allow others who DID add an extended-properties to THEN opt out of the fileset/xpaths (seemingly a long time ago actually). 

But yeah, thats the name.